src/assets.ts

```diff
@@ -327,7 +325,7 @@ function getBuildTasks(tasksConfiguration: tasks.TaskConfiguration): tasks.TaskD
 
     function findBuildTask(tasksDescriptions: tasks.TaskDescription[]) {
```

Comment:
What is VSCode's behavior if the user has an old style tasks.json? Should we be ignoring their tasks.json, replacing it, or is it really okay to add a new style task to an old style file?

Reply:
@gregg-miskelly The current behavior if a user has 0.1.0 in the version field of tasks.json is only an issue if they decide to use workspace folders.
![image](https://user-images.githubusercontent.com/3953714/32184502-a65410ae-bd59-11e7-9552-8224fb241ec1.png)

Therefore, if we see an old tasks.json that is nonempty, we should be ignoring it. We could add a warning message saying to upgrade it if they wish to use workspace folders?